cpp/src/Slice/Parser.cpp

```diff
@@ -2400,12 +2351,6 @@ Slice::Module::Module(const ContainerPtr& container, const string& name)
 // Constructed
 // ----------------------------------------------------------------------
 
```

Comment:
The `typeid` function is trivial and only used in 2 places, I inlined it in those 2 places.